src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -93,17 +94,28 @@ function ReportActionItemMessageEdit(props) {
     const {translate} = useLocalize();
     const {isKeyboardShown} = useKeyboardState();
     const {isSmallScreenWidth} = useWindowDimensions();
+    const isMobileSafari = Browser.isMobileSafari();
```

Comment:
Can we move it outside the functional component? It won't change in runtime. Simmilar approach can be seen in `ComposerWithSuggestions.js` file

Reply:
Thanks for this insight moved it outside the component.